imagely.es6

```diff
@@ -165,6 +180,50 @@ class Imagely {
 	}
 
 	/**
+	 * Assuming css, html, and js are the same in source files, render pages with unique json data.
+	 * The hope is
+	 *		1) We can make time gains by not opening and closing phantom every image.
+	 *		2) We can make time gains by caching repeated generated css, js, and html.
+	 *
+	 * @param {Object} phantomjs
+	 * @param {String} html Html to inject into the page.
+	 * @param {Object} page phantomjs page instance.
+	 * @param {Object} json The object we're looping over and setting uniquely per page.
+	 */
+	batch(phantomjs, html, page, json) {
+		console.info(`image ${this.batchIndex} of ${this.batchLength}`);
```

Comment:
We should avoid console logging. This may even break our usage of it in NBS since we use the console output of imagely.